package.json

```diff
@@ -25,9 +25,12 @@
     "hooks": {
       "post-merge": "post-npm-install",
       "post-rebase": "post-npm-install",
-      "pre-commit": "npm run lint"
+      "pre-commit": "lint-staged"
     }
   },
+  "lint-staged": {
+    "*.md": "markdownlint"
```

Comment:
Does this match all Markdown files, regardless of how deep they are within subdirectories?

Reply:
I can recheck, but AFAIK yes.

Reply:
Yeah it works.

Reply:
https://www.npmjs.com/package/lint-staged#filtering-files